apis/kueue/v1beta1/localqueue_types.go

```diff
@@ -149,6 +155,10 @@ type LocalQueueStatus struct {
 	// +kubebuilder:validation:MaxItems=16
 	// +optional
 	Flavors []LocalQueueFlavorStatus `json:"flavors,omitempty"`
+
+	// FairSharing contains the information about the current status of fair sharing.
+	// +optional
+	FairSharingStatus FairSharingStatus `json:"fairSharingStatus,omitempty"`
```

Comment:
```suggestion
	FairSharing FairSharingStatus `json:"fairSharing,omitempty"`
```
To make it consistent with CQ: https://github.com/kubernetes-sigs/kueue/pull/4687/files#diff-fa7799c26041d219ab624bc2069198280537236008ff41d199593618b900dcb9L314
Wdyt?

Reply:
Also, I think this should be *FairSharingStatus

Reply:
Should the link redirect me to something specific? Either way, I'm fine with changing that

Reply:
Done

Reply:
I think this is not adressed fully

Reply:
```
Add the field to the API type:

ensure the field is optional
add the omitempty struct tag
add the // +optional comment tag
add the // +featureGate=<gate-name> comment tag
ensure the field is entirely absent from API responses when empty (optional fields must be pointers)
```

Reply:
So, IIUC, new fields should always be ptrs, with omitempty

Reply:
Oh I had a stale version of your comment, will fix